docs/notebooks/bayesian_t-test.ipynb

```diff
@@ -38,14 +38,17 @@
    "cell_type": "markdown",
```

Comment:
<p>use singular for posterior?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I'm sorry I don't understand. What do you mean with singular?</p>

Reply:
<p>posterior vs posterior<strong>s</strong></p>